htdocs/core/class/commonobject.class.php

```diff
@@ -3884,6 +3882,19 @@ public function add_object_linked($origin = null, $origin_id = null, $f_user = n
 		}
 	}
 
+	/**
+	 * return element type string formated like element_element target_type and source_type
+	 * @return string
+	 */
+	public function getElementType()
+	{
+		// Elements of the core modules which have `$module` property but may to which we don't want to prefix module part to the element name for finding the linked object in llx_element_element.
+		// It's because an entry for this element may be exist in llx_element_element before this modification (version <=14.2) and ave named only with their element name in fk_source or fk_target.
```

Comment:
```suggestion
		// Elements of the core modules having a `$module` property but for which we may not not want to prefix the element name with the module name for finding the linked object in llx_element_element.
		// It's because existing llx_element_element entries inserted prior to this modification (version <=14.2) may already use the element name alone in fk_source or fk_target (without the module name prefix).
```